cluster/charts/rook-ceph/values.yaml

```diff
@@ -244,18 +247,30 @@ csi:
   forceCephFSKernelClient: true
   #rbdLivenessMetricsPort: 9080
   #kubeletDirPath: /var/lib/kubelet
-  #cephcsi:
-    #image: quay.io/cephcsi/cephcsi:v3.1.2
-  #registrar:
-    #image: k8s.gcr.io/sig-storage/csi-node-driver-registrar:v2.0.1
-  #provisioner:
-    #image: k8s.gcr.io/sig-storage/csi-provisioner:v2.0.0
-  #snapshotter:
-    #image: k8s.gcr.io/sig-storage/csi-snapshotter:v3.0.0
-  #attacher:
-    #image: k8s.gcr.io/sig-storage/csi-attacher:v3.0.0
-  #resizer:
-    #image: k8s.gcr.io/sig-storage/csi-resizer:v1.0.0
+  cephcsi:
+    image: # in image.registry.quay
+      repository: quay.io/cephcsi/cephcsi
```

Comment:
I don't think `repository` is the best name for this field, as the intended use actually calls for both repository and image.


components of an image are: `<registry>/<repo>/<image>:<tag>`

Considerations:
- should `registry` also include the `repo` and this be renamed to `image`?
- should there be a separate `repo` and `image` field?
- other?